pkgs/top-level/all-packages.nix

```
@@ -14398,8 +14398,6 @@ in
 
   sundials = callPackage ../development/libraries/sundials { };
```

@flokli perhaps in order to avoid merge conflicts with #79864 , maybe we should switch the python version of `sundials` uses _here_ instead of in #79864?

```suggestion
  sundials = callPackage ../development/libraries/sundials {
    python = python3;
  };
```

---

It's unrelated to the package bump - I'd rather see that in a separate commit in #79864.